cmd/evm/internal/t8ntool/execution.go

```diff
@@ -343,7 +343,7 @@ func (pre *Prestate) Apply(vmConfig vm.Config, chainConfig *params.ChainConfig,
 		// Amount is in gwei, turn into wei
 		amount := new(big.Int).Mul(new(big.Int).SetUint64(w.Amount), big.NewInt(params.GWei))
 		statedb.AddBalance(w.Address, amount)
-		statedb.Witness().TouchFullAccount(w.Address[:], true)
+		statedb.Witness().TouchFullAccount(w.Address[:], true, nil)
```

Comment:
For any `Touch*(...)` call that doesn't care about charging gas, we send the `nil` `UseGas(...)` function. 

This is the equivalent action of the previous API where we ignored the returned value (since we didn't charge that gas to anyone).